library/registry

```diff
@@ -1,8 +1,12 @@
 Maintainers: Milos Gajdos (@milosgajdos),
              Sebastiaan van Stijn (@thajeztah)
 GitRepo: https://github.com/docker/distribution-library-image.git
+GitFetch: refs/heads/master
+GitCommit: fa2dfdbfe65663972eaf9882447203c71f4ccb12
 
 Tags: 2.8.1, 2.8, 2, latest
 Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
-GitFetch: refs/heads/master
-GitCommit: 0be0d08b29d56bb1ef0fab93c751ca92d6976a19
+
+Tags: 2.8.2-beta.2
```

Comment:
Not sure if this should have any aliases?  Probably not for now, since a GA is expected tomorrow? :+1:

Reply:
Good one; did we have any before? `:rc` could work, but not sure if it's worth bothering

Reply:
```console
latest
2.8.1
2.8
2
2.8.0
2.8.0-beta.1
2.7.1
2.7
2.6.2
2.6
2.5.2
2.5
2.7.0
2.6.1
2.6.1-rc.2
2.6.0
2.6.0-rc.2
2.5.1
2.6.0-rc.1
2.5.0
2.5.0-rc.2
2.4.1
2.4
2.5.0-rc.1
2.4.0
2.3.1
2.3
2.3.0
2.2.1
2.2
0.9.1
0.8.1
2.2.0
0.6.1
0.6.6
0.6.8
0.6.9
0.7.1
0.7.0
0.8.0
0.6.0
0.9.0
0.7.3
0.6.2
0.7.2
0.5.9
0.6.5
0.6.3
0.6.7
0.6.4
2.1.1
2.1
2.0
2.0.0
2.0.1
```

Nope!